BLSQ/iaso
# IA-2566: Embedded links - user field : Value not found when the current user has not user permission

Explain what problem this PR is resolving
- When the current user has no **User management - Admin** the userSelect in embedded link display **a no found error**
- This ticket is about to fix that
Related JIRA tickets : [IA-2566](https://bluesquare.atlassian.net/browse/IA-2566)

## Self proofreading checklist


## Changes
- Give the possibility to define the value of managedUsersOnly parameter instead of the predefined "true" value
## How to test
- Remove the **User management - Admin** permission from one user
- Connect with the user and try to create a new embedded link or edit one and check if there is no an error in userSelect field
## Print screen / video
[Screencast from 2024-01-08 10-06-40.webm](https://github.com/BLSQ/iaso/assets/19631540/ed5ae061-ead3-4056-b22e-d733b77a4d3c)



[IA-2566]: https://bluesquare.atlassian.net/browse/IA-2566?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
